discover/portal/articles/001-content-sites/03-creating-and-styling-content/00-creating-content-intro.markdown

```diff
@@ -0,0 +1,12 @@
+# Creating Content
+
+So far you've created the Lunar Resort site's outline. By creating pages and
+using templates, the site now has a structure and the pages have layouts. Now
+it's time to color everything in with content.
+
+In this section you'll create content, some with more complex web content
+structures and templates, and use Liferay's Web Experience Management to publish
+your new content on your site.
+
+![Figure x: IMAGE.](../../images/.png)
```

Comment:
This is still missing the image. Anybody got an image they can commit? @stephenkostas @ngaskill